index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+	<head>
+
+		<title>Resume</title>
+		<meta charset="utf-8">
+
+	</head>
+	
+	<body>
+		<h1 align="center">Resume</h1>
```

Comment:
Recommendation - configure your Notepad or another code editor you use. Use 2 spaces for intentation instead of 4 or 8, like you use everywhere